src/libs/Launcher.js

```diff
@@ -169,17 +189,71 @@ export default class Launcher {
         .updateAttributes(folderId, { name: sourceAccountIdentifier })
     } catch (err) {
       log.warn(
-        `Could not rename the destination folder ${folderId} to ${sourceAccountIdentifier}. ${err.message}`
+        `Could not rename the destination folder ${folderId} to ${sourceAccountIdentifier}. ${err}`
       )
     }
   }
 
+  /**
+   * Ensures that account and triggers are created and launch the trigger
+   */
+  async ensureAccountTriggerAndLaunch() {
+    const startContext = this.getStartContext()
+    let { trigger, account, connector, client, job, launcherClient } =
```

Comment:
please, make us a favor and make a PR to change `connector` to `konnector` every where in this project. Having both is confusing ...

Reply:
https://github.com/cozy/cozy-flagship-app/pull/617